app/helpers/dashboard_helper.rb

```diff
@@ -8,4 +8,20 @@ def start_button_text(user)
       "Start now"
     end
   end
+
+  def start_button_path(user)
+    if user.cip_change_message.present?
+      user.participant_profile.guidance_seen? ? cip_change_alert_start_path : cip_path(user.core_induction_programme)
+    else
+      user.participant_profile.guidance_seen? ? cip_path(user.core_induction_programme) : guidance_question_path
+    end
+  end
```

Comment:
If user didn't see the guidance, we will never make the new `cip_change_message` for them, right?

So if the message is present, we know they saw the guidance, and therefore should go to `cip_change_alert_start_path` without checking anything.

Am I missing anything?

Reply:
ok, yeah that's right, i hadn't read the user sync properly enough.

Reply:
i'll switch it back.